src/workers/atlas_worker.js

```diff
@@ -22,6 +22,7 @@ export default class AtlasWorker extends PeriodicWorker {
     await this.dataModelEngine.downloadBundle(bundleId, sheltererId);
     await this.challengesRepository.resolveChallenge(challengeId);
     this.logger.info({message: '🍾 Yahoo! The bundle is ours.', bundleId});
+    await this.dataModelEngine.cleanupBundles();
```

Comment:
For challenge resolution worker, key thing to be effective is to be fast. So I assume, that this periodic worker should have very tight interval. At the same time `cleanupBundles()` executes two potentially very big calls to DB. Will it be performant enough? And if not, can we do something with this?